bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/tHq_4f_ckm_NLO_FXFX_MH125_CP/tHq_4f_ckm_NLO_FXFX_MH125_CP_proc_card.dat

```diff
@@ -0,0 +1,7 @@
+import model loop_sm-ckm
```

Comment:
you don;t need this model since its parameters will be overwritten by the model you import later

Reply:
thanks, has been removed.